### ISSUE TITLE
When option is 0, bulk update fails

### ISSUE THREAD

**Issue by cyyeoh** [rxcod9/joy-voyager-bulk-update#7]
When bulk updating the status column in my table which is radio button with 0 and 1, if I choose the option 0, the bulk update will fail.
This is because of what happens at the file path below around Line 36-42
vendor/joy/voyager-bulk-update/src/Http/Traits/BulkUpdateAction.php
At that code section, any 0 value is treated as a false result causing it to fail updating the table

**Comment by Ivanfut7** [comment]
Yes, same if tried to update a dropdown, if you change to some value can't put again value 0

**Comment by Ivanfut7** [comment]
Some update?

**Comment by cyyeoh** [comment]
no update, was just raising the issue to be known

**Comment by rxcod9** [comment]
Thank you for pointing it.
You can raise a PR.
